spec/requests/v1/saphyr/libraries_spec.rb

```diff
@@ -228,7 +232,8 @@
   context '#destroy' do
     context 'on success' do
       let!(:library) { create(:saphyr_library) }
-      let!(:tube) { create(:tube, material: library)}
+      let!(:tube) { create(:tube)}
+      let!(:container_material) { create(:container_material, container: tube, material: library)}
```

Comment:
You do this a few times: create the tube them assign a material to it be creating a new container material. Would it be worth adding a `tube_with_material` factory method that takes a transient material and creates a container material? The existing tube factory methods, e.g. `tube_with_saphyr_request` could then subclass this maybe, passing in the correct material?